packages/core/src/components/OverflowTooltip/OverflowTooltip.tsx

```diff
@@ -40,6 +40,10 @@ export const OverflowTooltip = (props: Props) => {
     setHover(truncated);
   };
 
+  if (!props.text) {
+    return null;
```

Comment:
Tbh I'm wondering if this is actually what you'd expect here. The `TextTruncate` component ends up defaulting `text` to `''` if it's not set, perhaps we should change the `Tooltip` `title` to do the same instead? I'm thinking you might get a bit of UI flicker if switching between `null` and text compared to `''` and text.